aslam_optimizer/sparse_block_matrix/cmake/Config.cmake.in

```diff
@@ -0,0 +1,16 @@
+@PACKAGE_INIT@
+
+include(CMakeFindDependencyMacro)
+find_dependency(Boost COMPONENTS system)
+find_dependency(Eigen3)
+find_dependency(sm_common)
+find_dependency(sm_eigen)
+
+list(APPEND CMAKE_MODULE_PATH ${CMAKE_CURRENT_LIST_DIR})
```

Comment:
@frneer is this necessary for `colcon` to tie them up? It's not good practice, it'll break downstream users on package release.